kong/dns/README.md

```diff
@@ -0,0 +1,139 @@
+Name
+====
+
+The module is currently Kong only, and builds on top of the `lua-resty-dns` and kong's `lua-resty-mlcache` library.
+
+Table of Contents
+=================
+
+* [Name](#name)
+* [APIs](#apis)
+    * [new](#new)
+    * [resolve](#resolve)
+    * [resolve_address](#resolve_address)
+
+# APIs
```

Comment:
I know why it is  the confusing in this section for you. Because they are very similar to original APIs 😄 . So they are not deprecated APIs, all of them are new ones.

(BTW, if the new library could be stable enough in the futher, we will deprecated the original dns client, then these APIs will exist independently, which no longer be confusing:)

Reply:
Also note that it serves the maintainers/developers of the dns client, so it will focus on two main thing here:
1. what are these APIs
2. how are these APIs implemented internally

Reply:
Yes, that was it 😄
Thank you